scripts/RunProfMPHPUnitTests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# This script runs the Sugar PHPUnit tests. It assumes the Sugar Docker stack has already been started and that
```

Comment:
This runs "your" tests, not the Sugar tests, right?

Reply:
Good catch!